entity-management-web/src/main/java/eu/europeana/entitymanagement/serialization/JsonLdSerializer.java

```diff
@@ -1,54 +1,68 @@
 package eu.europeana.entitymanagement.serialization;
 
+import static eu.europeana.entitymanagement.common.config.AppConfigConstants.BEAN_JSON_MAPPER;
+
 import java.io.IOException;
 import java.io.StringWriter;
 import java.text.SimpleDateFormat;
 import java.util.Locale;
 
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
 import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
 
+import eu.europeana.corelib.edm.model.schemaorg.Text;
 import eu.europeana.entitymanagement.common.config.AppConfigConstants;
 import eu.europeana.entitymanagement.definitions.exceptions.EntityManagementRuntimeException;
 import eu.europeana.entitymanagement.definitions.model.EntityRecord;
 import eu.europeana.entitymanagement.vocabulary.EntityProfile;
+import eu.europeana.entitymanagement.vocabulary.FormatTypes;
 
 @Component(AppConfigConstants.BEAN_EM_JSONLD_SERIALIZER)
 public class JsonLdSerializer {
 
-    ObjectMapper mapper;
-    public static final String DATE_FORMAT  = "yyyy-MM-dd'T'HH:mm:ss'Z'";
+    private final ObjectMapper mapper;
+    
+	public static final String DATE_FORMAT  = "yyyy-MM-dd'T'HH:mm:ss'Z'";
     
-    public JsonLdSerializer() {
-	mapper = new ObjectMapper();
-	SimpleDateFormat df = new SimpleDateFormat(DATE_FORMAT, Locale.ENGLISH);
-	mapper.setDateFormat(df);
-	
-	/*
-	 * necessary because of the @JsonFilter annotations if we want to include all fields,
-	 * otherwise the fields we want to exclude can be specified in the given FilerProvider
-	 */
-	
-	SimpleFilterProvider dummy = new SimpleFilterProvider();
-    dummy.setFailOnUnknownId(false);
-    mapper.setFilterProvider(dummy);
+    public JsonLdSerializer(@Qualifier(BEAN_JSON_MAPPER) ObjectMapper objectMapper) {
+		mapper = objectMapper.copy();
+		SimpleModule module = new SimpleModule();
+		/*
+		 * TODO: change the TextSerializer constructor in the corelib module to be public
```

Comment:
This change needs to be made in the API-Commons module

Reply:
I commented above.